views/event-form.hbs

```diff
@@ -0,0 +1,190 @@
+  </div>
+  <div>
+    <label for="disabled-parking">{{localText.disabledParking}}</label>
+    <input id="disabled-parking" type="checkbox" name="accessibility" value="Disabled parking" {{checkOptionsValue event.accessibilityOptions "Disabled parking"}}>
+  </div>
+  <div>
+    <label for="disabled-toilets">{{localText.disabledToilet}}</label>
+    <input id="disabled-toilets" type="checkbox" name="accessibility" value="Disabled toilets" {{checkOptionsValue event.accessibilityOptions "Disabled parking"}}>
+  </div>
+</fieldset>
+
+<fieldset>
+  <legend><span style="color:red;">*</span>{{localText.categories}}</legend>
+  <div>
+    <label for="competition">{{localText.competition}}</label>
+    <input id="competition" type="checkbox" name="categories" value="competition" {{checkOptionsValue event.categories "competition"}}>
+  </div>
+  <div>
+    <label for="music-venue">{{localText.conference}}</label>
+    <input id="conference" type="checkbox" name="categories" value="conference" {{checkOptionsValue event.categories "conference"}}>
+  </div>
+  <div>
+    <label for="culture">{{localText.culture}}</label>
+    <input id="culture" type="checkbox" name="categories" value="culture" {{checkOptionsValue event.categories "culture"}}>
+  </div>
+  <div>
+    <label for="dining">{{localText.dining}}</label>
+    <input id="dining" type="checkbox" name="categories" value="dining" {{checkOptionsValue event.categories "dining"}}>
+  </div>
+  <div>
+    <label for="educational">{{localText.educational}}</label>
+    <input id="educational" type="checkbox" name="categories" value="educational" {{checkOptionsValue event.categories "educational"}}>
+  </div>
+  <div>
+    <label for="launch">{{localText.launch}}</label>
+    <input id="launch" type="checkbox" name="categories" value="launch" {{checkOptionsValue event.categories "launch"}}>
+  </div>
+  <div>
+    <label for="music">{{localText.music}}</label>
+    <input id="music" type="checkbox" name="categories" value="music" {{checkOptionsValue event.categories "music"}}>
+  </div>
+  <div>
+    <label for="miscellaneous">{{localText.miscellaneous}}</label>
+    <input id="miscellaneous" type="checkbox" name="categories" value="miscellaneous" {{checkOptionsValue event.categories "miscellaneous"}}>
+  </div>
+  <div>
+    <label for="party">{{localText.party}}</label>
+    <input id="party" type="checkbox" name="categories" value="party" {{checkOptionsValue event.categories "party"}}>
+  </div>
+  <div>
+    <label for="sport">{{localText.sportEvent}}</label>
+    <input id="sport" type="checkbox" name="categories" value="sport" {{checkOptionsValue event.categories "sport"}}>
+  </div>
+  <div>
+    <label for="wedding">{{localText.wedding}}</label>
+    <input id="wedding" type="checkbox" name="categories" value="wedding" {{checkOptionsValue event.categories "wedding"}}>
+  </div>
+</fieldset>
+
+<span class="text-muted"><em><span style="color:red;">*</span>Required field</em></span>
```

Comment:
inline styling and 'Required field' should be from the `localText` file

Reply:
Also what is the "text-muted" class - am I being stupid, I can't see it in the css file?